### PR TITLE
Replace all instances of LoRa Server with ChirpStack

### LINKED ISSUE #345: Update Lora Server to ChirpStack in docs

### PR DESCRIPTION
**Pull request by luketainton**
#### What does this PR do?
This PR replaces all instances of 'LoRa Server' in the docs with 'ChirpStack'.

#### Do you have any concerns with this PR?
No.

#### How can the reviewer verify this PR?
Check through the changes to ensure they are happy with all replacements.

#### Any background context you want to provide?
Fixes #345 

#### Questions:
- Have you connected this PR to the issue it resolves?
Yes - #345 
- Does the documentation need an update?
This PR is updating the docs.
- Does this add new dependencies?
No.
- Have you added unit or functional tests for this PR?
Not neccessary.

### CODE REVIEW

File: docs/guides/getting-started.md [hunk 1]
```diff
@@ -53,8 +53,8 @@ Running the demo is the easiest way to try out LPWAN Server.
 
 - LPWAN Server REST API - https://localhost:3200/api
 - LPWAN Server Web Client - https://localhost:3200
-- LoRa App Server - https://localhost:8082
-- LoRa App Server V1 - https://localhost:8081
+- ChirpStack LoRa App Server - https://localhost:8082
+- ChirpStack LoRa App Server V1 - https://localhost:8081
```
Comment (rhythnic) [inline docs/guides/getting-started.md]: remove "LoRa"

**Comment by luketainton** [comment]
Hi @rhythnic ! I've implemented these changes for you - please check through them to make sure I haven't missed anything :)

**Comment by rhythnic** [comment]
I forgot that we'll need to update "Lora Server" to "Chirpstack" in the tests.  If you do a search/replace, case insensitive, from "Lora Server" to "Chirpstack" for the whole test directory, it should fix it.  If not, update the PR to merge into `issue-345` and I will take care of running/fixing the tests locally.  Thanks for all your help.

**Comment by rhythnic** [comment]
Thanks @luketainton .  I'll merge this tomorrow and publish the new documentation.

**Comment by luketainton** [comment]
Always happy to help @rhythnic 👍

**Pull request merged**